pom.xml

```diff
@@ -84,5 +92,17 @@
             <version>1.0</version>
             <systemPath>${project.basedir}/src/main/resources/calculator-1.0.jar</systemPath>
         </dependency>
+        <!-- https://mvnrepository.com/artifact/io.github.bonigarcia/webdrivermanager -->
```

Comment:
Why did you add this comment here?

Reply:
I copied it from the webdriver manager documentation.